src/main.rs

```diff
@@ -58,3 +55,10 @@ fn panic(info: &PanicInfo) -> ! {
         x86_64::instructions::hlt();
     }
 }
+
+fn abort() -> ! {
+    x86_64::instructions::interrupts::disable();
```

Comment:
doesn't really matter but it's better if it's inside the loop in case something weird like an NMI happens